packages/devtools_app/lib/src/service_manager.dart

```diff
@@ -1322,61 +1329,6 @@ class ServiceExtensionState {
       );
 }
 
-class VmFlagManager extends Disposer {
```

Comment:
moved this manager to the existing `vm_flags.dart` file so that `service_manager.dart` has more logically grouped classes.